doc/tools/coqrst/coqdomain.py

```diff
@@ -1021,6 +1026,10 @@ def parse_options(node):
         opt_input = 'in' in options
         opt_output = 'out' in options
 
+        # if 'extra' is given and env variable 'COQRST_EXTRA' is set,
+        # allow errors
+        opt_fail = opt_fail or (opt_extra and os.environ.get('COQRST_EXTRA', '0') == '0')
```

Comment:
This logic seems incorrect. Errors should be allowed if `fail` is given or if `extra` is given and `COQRST_EXTRA` is not set (or set to `0`).

Reply:
In fact, the logic was good but the above comment was wrong.